Readme.md

```diff
@@ -87,39 +87,10 @@ application bundle using the following steps:
 
 Doing this is more complex as it requires installation of library dependencies for both x64 and ARM (or universal library
 equivalents) and may require specifying additional arguments to point to relevant library locations. 
-Execute BuildMacOSUniversalBinary.py --help for more details.  
+Execute BuildMacOSUniversalBinary.py --help for more details.
 
-### Linux Global Build Steps:
-
-To install to your system.
-
-1. `mkdir build`
-2. `cd build`
-3. `cmake ..`
-4. `make -j $(nproc)`
-5. `sudo make install`
-
-### Linux Local Build Steps:
-
-Useful for development as root access is not required.
-
-1. `mkdir Build`
-2. `cd Build`
-3. `cmake .. -DLINUX_LOCAL_DEV=true`
-4. `make -j $(nproc)`
-5. `ln -s ../../Data/Sys Binaries/`
-
-### Linux Portable Build Steps:
-
-Can be stored on external storage and used on different Linux systems.
-Or useful for having multiple distinct Dolphin setups for testing/development/TAS.
-
-1. `mkdir Build`
-2. `cd Build`
-3. `cmake .. -DLINUX_LOCAL_DEV=true`
-4. `make -j $(nproc)`
-5. `cp -r ../Data/Sys/ Binaries/`
-6. `touch Binaries/portable.txt`
+## Building for Linux
+Refer to the [wiki](https://github.com/dolphin-emu/dolphin/wiki/Building-for-Linux) for distribution-specific instructions.
```

Comment:
I don't know if this needs to say "distribution-specific". Other than that, LGTM.